src/app/services/local-storage.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LocalStorageService {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  setLSItem(key: string, value: string | { [key: string]: any }): void {
```

Comment:
{ [key: string]: any } has a name, it is Record<string, any>. You can read about typescript types in https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype

Reply:
Done